IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using IndicoV2.Submissions.Models;
+
+namespace IndicoV2.Submissions
+{
+    public interface ISubmissionsClient
+    {
+        Task<IEnumerable<int>> CreateAsync(int workflowId, Stream[] streams, CancellationToken cancellationToken = default);
```

Comment:
This should be part of workflows. We submit files to a workflow that creates a submission, so SubmitToWorkflow or something along the lines can be more appropriate. We can even have something like SubmitFilesToWorkflow and SubmitStreamsToWorkflow etc. to distinguish between them.

Reply:
In Indico's graphql schema there's type `Submission` with it's own Id, there's query `submissions` returning `Submission` type,  and the mutation to create `Submission` is called `WorkflowSubmission`. There's also a separate type called `Workflow`. 
Because operation is creating `Submission` type I'd rather leave it in `ISubmissionsClient` (`IWorkflowClient` suppose to be handling `Workflow` type).

About renaming: 
- Stream/string part is being handled by polymorphism here, so adding this part would be redundant
- `Submit` vs `Create` - I have thought that ISubmissionClient.Submit would be too much, but I'm OK with this change.

Reply:
This can be a issue to be consistent with the meaning and api from other clients. For more reference, take a look at https://github.com/IndicoDataSolutions/indico-client-python/tree/master/indico/queries
Submitting files to a workflow should be associated with a workflow because only a workflow or a workflow id is required to submit files/urls.